composer.json

```diff
@@ -7,7 +7,7 @@
   ],
   "require": {
     "neos/neos": "^8.3.0",
-    "neos/neos-ui-compiled": "self.version"
+    "neos/neos-ui-compiled": "8.4.x-dev"
```

Comment:
Revert this!

Reply:
You could have probably also did this override from your root composer.json.
I don' remember having this issue.